docs/patterns/frontend/page.md

```
@@ -15,395 +15,24 @@ apps/web/src/app/blog/[id]/page.tsx         # /blog/123
-  <aside className="fixed top-30 right-[max(1rem,calc(50%-38rem))] hidden w-40 xl:block">
-    {/* ToC 등 */}
-  </aside>
-
-  {/* 메인 콘텐츠 */}
-  <main className="max-w-4xl">
-    {/* 내용 */}
-  </main>
-</div>
-```
+| 상황 | 참조 파일 |
+|------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)